File: output/ChatGPT/codenet/Java/C++/s321983475.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int w, a, b;
    cin >> w >> a >> b;
    
    int answer = max(a, b) - min(a, b) - w;
    if(answer < 0){
        cout << 0 << endl;
    }else{
        cout << answer << endl;
    }
    
    
    return 0;
}

File: output/CodeGen/codenet/Java/Python/s321983475.py
def main():
    w = int(input())
    a = int(input())
    b = int(input())
    
    answer = max(a,b)-min(a,b) -w
    if answer < 0:
        print(0)
    else:
        print(answer)

if __name__ == "__main__":
    main()